desktop/api/src/main/java/org/datacleaner/panels/MutableInputColumnListPanel.java

```diff
@@ -51,7 +52,18 @@
     private final AnalysisJobBuilder _analysisJobBuilder;
     private final OutputColumnVisibilityButton _visibilityButton;
 
-    public MutableInputColumnListPanel(AnalysisJobBuilder analysisJobBuilder, MutableInputColumn<?> inputColumn) {
+    public JComponent panelOwner;
```

Comment:
fixed - I forgot the field setting. Was caused by copy&paste from project where I tried it. Not good approach...